elements/urth-core-behaviors/general-behavior.html

```diff
@@ -0,0 +1,51 @@
+<!--
+# Copyright (c) Jupyter Development Team.
+# Distributed under the terms of the Modified BSD License.
+-->
+
+<!--
+This behavior is used to encapsulate some of the services needed for urth elements.
+-->
+<link rel="import" href="shared-styles.html">
+<style is="custom-style" include="shared-styles"></style>
+<script>
+    'use strict';
+    (function() {
+        window.Urth = window.Urth || {};
+
+        /**
+         * Behavior that encapsulate some of the functions needed by
+         * both urth-core and urth-viz elements.
+         *
+         * @polymerBehavior Urth.GeneralBehavior
+         */
+        Urth.GeneralBehavior = {
```

Comment:
Should have mentioned this earlier, but `GeneralBehavior` is not a good name for this. Why not call it `ErrorReportingBehavior`?